avm/res/kubernetes-configuration/extension/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,84 @@
+targetScope = 'subscription'
+
```

Comment:
metadata name = 'WAF-aligned'
metadata description = 'This instance deploys the module in alignment with the best-pratices of the Azure Well-Architected Framework.'